pkg/grizzly/config.go

```diff
@@ -2,8 +2,8 @@ package grizzly
 
 // GrizzlyOpts contains options for all Grizzly commands
 type GrizzlyOpts struct {
-	Directory    *string
-	ResourceFile *string
+	ResourcePath *string
+	Directory    *bool
 	JsonnetPaths *[]string
 	Targets      *[]string
```

Comment:
Why are these pointers? slices are pointers anyways, so you double-pointer here, and the bool doesn't need to be one as well, right?

Also consider you risk panicking when using those variables without checking they are non nil. No need to use unsafe constructs here ;)